web/web_types/types.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2019 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package web_types
+
+import (
+	"net/url"
+	"time"
+)
+
+// RunRuntimeInfo contains runtime info about Prometheus.
+type RuntimeInfo struct {
+	Birth               time.Time          `json:"startTime"`
+	CWD                 string             `json:"cwd"`
+	Version             *PrometheusVersion `json:"version"`
+	Alertmanagers       []*url.URL         `json:"alertmanagers"`
+	GoroutineCount      int                `json:"goroutineCNT"`
+	GOMAXPROCS          int                `json:"goMaxProcs"`
+	GOGC                string             `json:"goGC"`
+	CorruptionCount     int64              `json:"corruptionCount"`
+	ChunkCount          int64              `json:"chunkCount"`
+	TimeSeriesCount     int64              `json:"timeSeriesCount"`
```

Comment:
Count here still implies a counter, which this isn't. And chunks, goroutines.

Reply:
I wouldn't have made that association because it's not a metric name, but don't care either way.

On the other hand you might ask if we really need to offer data over this API that we already have on /metrics as well (goroutines, number of time series, etc.).

Reply:
> On the other hand you might ask if we really need to offer data over this API that we already have on /metrics as well (goroutines, number of time series, etc.).

I feel like providing an API makes those a bit more accessible. 

Should I remove it?